docs/Server.md

```diff
@@ -134,10 +134,33 @@ interface by having the following methods: `info`, `error`, `debug`, `fatal`, `w
       return child;
     },
   };
-  
+
   const fastify = require('fastify')({logger: customLogger});
   ```
 
+<a name="factory-disable-request-logging"></a>
+### `disableRequestLogging`
+By default, when logging is enabled, Fastify will issue an `info` level log
+message when a request is received and when the response for that request has
+been sent. By setting this option to `true`, these log messages will be disabled.
+This allows for more flexible request start and end logging by attaching
+custom `onRequest` and `onResponse` hooks.
+
++ Default: `false`
+
+```js
+// Examples of hooks to replicate the disabled functionality.
+fastify.addHook('onRequest', (req, reply, next) => {
+  req.log.info({ url: req.req.url, id: req.id }, 'received request')
+  next()
+})
+
+fastify.addHook('onResponse', (req, reply, next) => {
+  req.log.info({ url: req.req.originalUrl, statusCode: res.res.statusCode }, 'request completed')
```

Comment:
`res.res.statusCode` should be `reply.res.statusCode` and `req.log.info` should be `reply.log.info` in next line.

Reply:
A PR to fix it is welcome.

Reply:
@jsumners Added you as reviewer [Here](https://github.com/fastify/fastify/pull/2099)